### ISSUE TITLE
RC1 Build

### ISSUE THREAD

**Issue by danny-waite** [justkao/Bolt#4]
Hey, great project! but I'm having a hell of a time trying to get things up and running on RC1.  I see the packages are not yet updated but you made various commits to get ready for RC1.  What's the current status? Any help on how to progress?

**Comment by justkao** [comment]
Hey Danny, thanks you for your interest ;) The features are pretty much locked, but it's still work in progress (mostly adapting to changes in ASP .NET 5 + performance tuning). What kind of issues are you experiencing ? I would be glad to help.

**Comment by danny-waite** [comment]
Thanks for the quick response, and offer to help.  Well currently I'm trying to get to a proof of concept so nothing too in depth.

The Bolt.Samples don't seem to work with RC1 and if I create a new ASPNET 5 Project in VS2015 and install the Bolt.Server package I guess errors about its dependencies

Unable to locate Dependency Microsoft.Framework.ActivatorUtilities.Sources >= 1.0.0-beta8-15657
Unable to locate Dependency Microsoft.Framework.DependencyInjection.Abstractions >= 1.0.0-beta8-15621
Unable to locate Dependency Microsoft.Framework.Caching.Abstractions >= 1.0.0-beta8-15635
Unable to locate Dependency Microsoft.Framework.Logging.Abstractions >= 1.0.0-beta8-15621
Unable to locate Dependency Microsoft.Framework.Configuration.Abstractions >= 1.0.0-beta8-15635
Unable to locate Dependency Microsoft.Framework.OptionsModel >= 1.0.0-beta8-15618

So in short, just need a working formula I can follow.

Thanks again.

**Comment by justkao** [comment]
The Bolt packages on Nuget are old. At this point i am not even sure that ASP .NET 5 - beta 8 is supported in visual studio. I need to upload Bolt packages build for RC1. I will try to do it by tomorrow and also update the sample projects. Then you should have no problem building the solutions.

**Comment by danny-waite** [comment]
What a guy! Thanks.

**Comment by justkao** [comment]
Hey Danny,
I have uploaded the actual version of Bolt to Nuget (0.19.0-alpha1). I have also added new sample project - "Bolt.Console.Sample" that works with this version. The other sample projects are not yet updated. Let me know if you have any issues running the sample.

Martin

**Comment by danny-waite** [comment]
Hey Martin,

I just created a new ASPNET 5 Project and added the Bolt.Server package, looks better now but still get an error on:

Unable to locate Dependency Microsoft.Extensions.ActivatorUtilities.Sources >= 1.0.0-rc1-final

I'll test the Sample now and let you know

**Comment by danny-waite** [comment]
Yeah seems same on the sample.

https://www.dropbox.com/s/gxtddnr3gb8ceto/Screenshot%202015-12-01%2011.37.45.png?dl=0

**Comment by justkao** [comment]
It seems that Microsoft.Extensions.ActivatorUtilities.Sources is not published on nuget server. To fix this I have uploaded new Bolt.Server.0.20.0-alpha1 package that does not have that dependency. 
The Sample project is also updated, I hope it will be working now.

**Comment by danny-waite** [comment]
Looking good now Martin, thanks! I'll have a play around and come back to you.
